error_manager.go

```diff
@@ -47,7 +47,11 @@ func (em *ErrorManager) getErrorMessage(input GenericComponentMsg) string {
 		inHash := c.(map[string]interface{})
 		status := inHash["status"].(string)
 		if status == "errored" {
-			return inHash["error"].(string)
+			err, ok := inHash["error"].(string)
+			if ok {
+				return err
+			}
+			return ""
```

Comment:
@purehyperbole in order to accomplish this is better just this:
```
str, _ := inHash["error"].(string)
return str
```

However would it be nice if we can return an `Internal error #XXXX` message and a code so we know exactly where the error is coming from